tests/Unit/Container/ContainerTest.php

```diff
@@ -4,11 +4,11 @@
 
 use Illuminate\Container\Container;
 use Mockery as m;
-use TestCase;
+use Mongolid\TestCase;
 
 class IocTest extends TestCase
 {
-    public function tearDown()
+    public function setUp(): void
```

Comment:
I don't know how it is working....

```suggestion
    public function tearDown(): void
```